### ISSUE TITLE
Pass in JAVA_OPTS Don't Work for Proxy

### ISSUE THREAD

**Issue by stevenmccord** [SeleniumHQ/docker-selenium#196]
I am utilizing the standalone-firefox-debug image as the base.  

I am trying to set some options for selenium as well as some java options.  The SE_OPTS fail hard and I can't start the container when I try to set a firefoxProfileTemplate and/or -log.  In addition, I noticed that the JAVA_OPTS don't fail, but I don't see them proxying anything either, so not sure if they are just failing softly or I am just doing something wrong (which certainly could be the problem.)

Here are the environment variables being set in the Dockerfile:

```
ENV SE_OPTS="firefoxProfileTemplate /home/ubuntu/project/ac9xlz80.functional-tests -log /home/ubuntu/project/selenium.log"

ENV JAVA_OPTS="-Dhttp.proxyHost=[IP] -Dhttp.proxyPort=8080 -Dhttp.proxyUser=[USER] -Dhttp.proxyPassword=[PWD]"
```

When I start with these options I get the following error:

```
Waiting xvfb...
-bash: /home/ubuntu/project/ac9xlz80.functional-tests: Is a directory
Waiting xvfb...
Waiting xvfb...
Waiting xvfb...
Waiting xvfb...
Waiting xvfb...
Waiting xvfb...
Waiting xvfb...
Waiting xvfb...
Waiting xvfb...
```

When I start the container it get the following error if I just set the log solo:

```
Waiting xvfb...
-bash: /home/ubuntu/project/selenium.log: No such file or directory
Waiting xvfb...
Waiting xvfb...
Waiting xvfb...
Waiting xvfb...
Waiting xvfb...
Waiting xvfb...
Waiting xvfb...
Waiting xvfb...
Waiting xvfb...
```

Here is what the environment variables look like in the selenium container

```
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
HOSTNAME=b547a8838768
TERM=xterm
DEBIAN_FRONTEND=noninteractive
DEBCONF_NONINTERACTIVE_SEEN=true
TZ=US/Pacific
SCREEN_WIDTH=1360
SCREEN_HEIGHT=1020
SCREEN_DEPTH=24
DISPLAY=:99.0
LANGUAGE=en_US.UTF-8
LANG=en_US.UTF-8
SE_OPTS=firefoxProfileTemplate /home/ubuntu/project/ac9xlz80.functional-tests -log /home/ubuntu/project/selenium.log
JAVA_OPTS=-Dhttp.proxyHost=[IP] -Dhttp.proxyPort=8080 -Dhttp.proxyUser=[USER] -Dhttp.proxyPassword=[USER]
HOME=/root
```

**Comment by stevenmccord** [comment]
More information here, it is happening here during the entry-point.sh script:

```bash
$(for E in $(grep -vxFf asseluser asroot); do echo $E=$(eval echo \$$E); done)
```

I think it has something to do with the fact that these options have a space in the environment variable so trying to update, but figured I would provide as much information as I can.  Thanks!

**Comment by stevenmccord** [comment]
Looks like I solved the profile issue that is solved by passing in this option:

`-Dwebdriver.firefox.profile=functional-profile`

However, I am still not understanding how to deal with an authenticated proxy, when I pass in those java options I still I am not able to proxy through anything, it actually doesn't do anything when I open up a mozilla browser, it is like the proxy didn't get set.  Any help there would be extremely helpful.  Thanks!

**Comment by ondrej** [comment]
@stevenmccord, take a look at #207, there's a proposed fix there that might help.

**Comment by tarantegui** [comment]
@ondrej your fix works for me on the debug version, but not on standalone. ¿Do yo have configured correctly a standalone-firefox with proxy?

**Comment by diemol** [comment]
This was solved with #459, which was merged and released, this issue can be closed.
